components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -881,9 +891,11 @@ subroutine ma_convproc_dp_intr(                &
             tmpa = hund_ovr_g*mu(ii,k) ; tmpb = hund_ovr_g*md(ii,k)
          end if
          if ((k > 15) .and. (mod(k,5) == 1)) write(lun,'(a)')
-         write(lun,'(a,i4,1p,4e11.3,5(2x,2e10.3))') 'qakp', k, &
-            state%zi(i,k+1), fracis(i,k,ixso4_a1), tmpa, tmpb, &
-            (qaa(i,k,l), (qbb(i,k,l)-qaa(i,k,l)), l=ixh2o2,ixh2o2+3)
+!++hybrown
```

Comment:
We should lose all these signature lines and commented out code

Reply:
Sounds good. I will go through and update.